File: Runners/JavaRunner/java-runner/src/main/java/com/mems/Shared/Models/TestCase.java
package com.mems.Shared.Models;

public class TestCase {
    public String name = "Test";
    public String testLanguage = "";
    public String testInitialization = "";
    public String inputExpression = "";
    public String outputExpression = "";

    public TestCase(){}

    public TestCase(String name, String testLanguage, String testInitialization, String inputExpression, String outputExpression) {
        this.name = name;
        this.testLanguage = testLanguage;
        this.testInitialization = testInitialization;
        this.inputExpression = inputExpression;
        this.outputExpression = outputExpression;
    }
}

File: Shared/Models/TestCase.cs
namespace Shared.Models
{
    public class TestCase
    {
        public string Name { get; set; } = "Test";
        public string TestLanguage { get; set; } = string.Empty;
        public string TestInitialization { get; set; } = string.Empty;
        public string InputExpression { get; set; } = string.Empty;
        public string OutputExpression { get; set; } = string.Empty;

        public TestCase()
        {
        }

        public TestCase(TestCase source)
        {
            Name = source.Name;
            TestLanguage = source.TestLanguage;
            TestInitialization = source.TestInitialization;
            InputExpression = source.InputExpression;
            OutputExpression = source.OutputExpression;
        }
    }
}
